githubscript/examples/config.example.json

```diff
@@ -1,15 +1,15 @@
 {
     "work_dir": "/tmp/githubscript/work_dir",
     "github_projects": {
-        "fenix": {
+        "reference-browser": {
           "allowed_actions": ["release"],
           "github_token": "SOME_TOKEN",
           "github_owner": "mozilla-mobile",
-          "github_repo_name": "firefox-android",
+          "github_repo_name": "reference-browser",
```

Comment:
reference-browser does not actually use githubscript. I've replaced firefox-android with reference-browser in examples and tests just to provide the name of an active github repo; open to suggestions for a better approach!

Reply:
Should we remove the mobile bits entirely and use an example from xpi instead?

Reply:
Maybe? I'm mostly reluctant because I'm not familiar with xpi or githubscript. Would that just be changing to mozilla-extensions/xpi-manifest/some different scope?